src/Sextant/Navigation/NavigationParameter.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) 2019 .NET Foundation and Contributors. All rights reserved.
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Sextant
+{
+#pragma warning disable CA1710, CA2237
```

Comment:
Use SuppressMessage attribute for these two guys.

Reply:
Added attributes